internal/arrowutil/vector_values.gen.go

```diff
@@ -0,0 +1,481 @@
+// Generated by tmpl
```

Comment:
It is possible to get this file into the values package?

Reply:
Yes. I originally put it here because I was using `arrowutil/array_values.gen.go` as a starting point for this, and it was convenient to just use the same template data. Now that I have something that works though, I'll move it to the values package.